website/conferences/views.py

```diff
@@ -28,6 +29,22 @@
 
 logger = logging.getLogger(__name__)
 
+# Setup Keen client
+keen.project_id = settings.KEEN_PROJECT_ID
+keen.write_key = settings.KEEN_WRITE_KEY
+
+
+def _dt_to_iso8601(value):
```

Comment:
@sloria, it would need millisecond support, but yes.  That function could use some timezone-verification, though.